.github/workflows/ci.yml

```diff
@@ -11,6 +11,10 @@ concurrency:
   group: ci-${{ github.head_ref || github.run_id }} # will be canceled on subsequent pushes in pull requests but not branches
```

Comment:
> IMHO distinction between "tests" and "static analysis" is not clear, because running Fixer in dry run mode is rather the latter, but is included in ci workflow under tests.

Agree, I would love to have them separated on long run. The status you described is part of master already, I love to separate it on long run, open for suggestion (or out of PR scope).
One with I discover is how to keep almost whole PHP/flags/envs setup same for `tests` and `fixer check` the same, yet to not duplicate the CI config. Started some extractions in this PR already, as you see - hope it will get us closer.